src/main/java/uk/gov/justice/hmpps/prison/service/CaseNoteService.java

```diff
@@ -121,7 +121,6 @@ public CaseNote getCaseNote(final Long bookingId, final Long caseNoteId) {
     }
 
     @Transactional
-    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER"})
```

Comment:
createCaseNote and updateCaseNote are now only called from methods in this file which have the VerifyOffenderAccess annotation - so this annotation can be removed